flink-streaming-scala/src/main/scala/org/apache/flink/streaming/api/scala/AllWindowedStream.scala

```diff
@@ -70,7 +71,7 @@ class AllWindowedStream[T, W <: Window](javaStream: JavaAllWStream[T, W]) {
    * thrown.
    */
   @PublicEvolving
-  def allowedLateness(lateness: Time): AllWindowedStream[T, W] = {
+  def allowedLateness(lateness: Duration): AllWindowedStream[T, W] = {
```

Comment:
There are scala changes like that that didn't include proper deprecation because we wanted to remove scala entirely, anyway, in 2.0.

What's the state with that?